css/component_styles/snackbar.css

```diff
@@ -0,0 +1,60 @@
+.container {
+    background-color: var(--primary-snackbar-color);
+    border-radius: 0.5rem;
+    font-size: 1.5rem;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0 1rem;
+    width: 30%;
+    margin: auto;
+    position: fixed;
+    left: 0;
+    right: 0;
+    bottom: 3rem;
+    box-shadow: 0 0 5px var(--primary-snackbar-color);
+    z-index: 999;
+    visibility: hidden;
+}
+
+.container p {
+    margin-right: auto;
+    color: var(--secondary-color);
+}
+
+.container button {
+    /* color: var(--btn-color); */
+    background-color: transparent;
+    border: none;
+    margin-right: 1rem;
+    font-weight: bold;
+}
+
+.exit i {
```

Comment:
Don't target or give CSS property directly to any Html element try to class and use it.

Reply:
Done

Reply:
Done👍🏻